src/sidebar/store/test/util-test.js

```diff
@@ -36,6 +51,15 @@ describe('reducer utils', function() {
   });
 
   describe('#createReducer', function() {
+    it('can not return undefined during initialization', function() {
```

Comment:
It wasn't immediately clear to me what this was trying to test. `it("returns an object if input state is undefined and action is unrecognized")` might be a more direct description, with a brief note on when this happens as a comment below.

Reply:
The action does not matter here, but perhaps using "fake" is misleading for that action